README.md

```diff
@@ -102,9 +105,11 @@ transaction = (
         .set_decimals(2)
         .set_initial_supply(1000)
         .set_treasury_account_id(operator_id)
+        .set_admin_key(admin_key)
```

Comment:
I'd might have a note here explaining that admin key is optional; it isn't needed to create a token per se, but if that token should have the ability to be deleted or updated it is needed to pass the key. 
Alternatively you could have 2 examples, one with an admin key and one without (also interesting to explain the change in signature/the double signing)